lib/CGI/Application/Plugin/Throttle.pm

```diff
@@ -211,23 +227,29 @@ sub _get_redis_key
 }
 
 
-=head2 count
 
-Return the number of times the remote client has hit a run mode, along
-with the maximum allowed visits:
+=head2 C<count>
 
-=for example begin
+Returns two values: the number of times the remote client has hit a run mode,
+along with the maximum allowed visits:
 
-      sub your_run_mode
-      {
-          my ($self) = (@_);
+=for example begin
 
-          my( $count, $max ) = $self->throttle()->count();
-          return( "$count visits seen - maximum is $max." );
-      }
+    sub your_run_mode
+    {
+        my ($self) = (@_);
+        
+        my( $count, $max ) = $self->throttle()->count();
+        return( "$count visits seen - maximum is $max." );
+    }
 
 =for example end
 
+=head3 warning
+
+This method must be called in list context, in scalar context, the result will
+always be '2'.
```

Comment:
I guess this is true, but it seems like an odd thing to note!

Reply:
I would not have mentioned this if `count` would just do that, give back the number of counts currently 'active', that is what every person would expect.

Had it been called `stats` one would expect multiple results, an array reference or a hash reference.

```
my $active = $self->throttle->count;
```
which would look natural, just does not that.

At the time when you created that, it was your chance to use `wantarray` but that is a passed station now.